ovos_PHAL_plugin_homeassistant/__init__.py

```diff
@@ -261,7 +261,6 @@ def init_configuration(self, message=None):
                     configuration_host,
                     configuration_api_key,
                     configuration_assist_only,
-                    self.max_ws_message_size
```

Comment:
This is only being dropped from the HomeAssistantRESTConnector, as it isn't used there